batch-setup/make_rawr_tiles.py

```diff
@@ -10,6 +10,16 @@
 import shutil
 import tempfile
 import yaml
+import sys
+import os
+from distutils.util import strtobool
+
+currentdir = os.path.dirname(os.path.realpath(__file__))
+parentdir = os.path.dirname(currentdir)
+sys.path.append(parentdir)
+
+from utils.tiles import BoundingBoxTilesCoordinateGenerator
+from utils.tiles import TilesCoordinateGenerator
```

Comment:
> are you using this import?

yup in the main function

Reply:
Oh I only saw the usage of the BoundingBox version

Reply:
> Oh I only saw the usage of the BoundingBox version

I don't think I understand this comment? are you saying we shouldn't import it globally? Importing globally vs inside function is for speed optimization, however in our case, I don't think it makes significant difference on the importing, by importing at the top it just improves the readability, later I will add a linter so that all the imports will be sorted.

Reply:
or maybe you mean TilesCoordinateGenerator? 
that is used in the typing hint

![Uploading Snip20210519_6.png…]()